modules/azure/src/test/java/org/testcontainers/azure/AzureServiceBusEmulatorContainerTest.java

```diff
@@ -0,0 +1,102 @@
+package org.testcontainers.azure;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusMessage;
+import com.azure.messaging.servicebus.ServiceBusProcessorClient;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+import com.azure.messaging.servicebus.ServiceBusSenderClient;
+import com.azure.messaging.servicebus.ServiceBusTransactionContext;
+import com.github.dockerjava.api.model.Capability;
+import org.assertj.core.api.Assertions;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.MSSQLServerContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AzureServiceBusEmulatorContainerTest {
+
+    @Rule
+    // network {
+    public Network network = Network.newNetwork();
+
+    // }
+
+    @Rule
+    // sqlContainer {
+    public MSSQLServerContainer<?> mssqlServerContainer = new MSSQLServerContainer<>(
+        "mcr.microsoft.com/mssql/server:2022-CU14-ubuntu-22.04"
+    )
```

Comment:
very cool to see it works with mssql too!

Reply:
Yeah, Azure SQL is 99% MS SQL 🙂